Dockerfile

```diff
@@ -5,6 +5,10 @@ RUN apt-get update
 
 # install dependencies needed for setting up R
 RUN apt-get install -y lsb-release dirmngr gnupg apt-transport-https ca-certificates software-properties-common
+RUN apt-get install -y libharfbuzz-dev libfribidi-dev
+
+# RUN apt-get -y build-dep libcurl4-gnutls-dev
```

Comment:
Do we need this here still?

Reply:
No, I will get rid of that commented bit.